checkbox-radio.scss

```diff
@@ -14,20 +14,20 @@
 	& > input[type="#{$type}"] {
 		margin-top: -0.2rem;
 		margin-right: 0.6rem;
-		margin-left: -1.7rem;
+		margin-left: -1.6rem;
```

Comment:
@dlockhart I think this change will move the checkbox farther to the left, which might be the opposite of what we want here. We expect checkboxes to be to the right of the label, right?